docs/init-repo.md

```diff
@@ -0,0 +1,38 @@
+# init-repo.ps1
+Repository initialization.
+
+Input parameters:
+* **$AzureDevOpsCollection**: url to Azure DevOps, e.g. https://dev.azure.com/oriflame/ (optional)
+* **$DevOpsProject**: name of the project in Azure DevOps, e.g. http://dev.azure.com/oriflame/{THIS}
+* **$RepositoryName**: name of the project in Azure DevOps, e.g. http://dev.azure.com/oriflame/something/_git/{THIS}
+
+It does following:
+
+* check & install prerequisities: 
+  * in case GIT command is missing it installs [chocolatey](https://chocolatey.org/) and [git](https://chocolatey.org/packages/git.install)
+  * in case TF tool is missing (C:\Program Files (x86)\Microsoft Visual Studio\2019\TeamExplorer\Common7\IDE\CommonExtensions\Microsoft\TeamFoundation\Team Explorer\tf.exe is searched) it installst [Team Explorer](https://chocolatey.org/packages/visualstudio2017teamexplorer)
+* it checks, if we are in GIT repo and if not, it clones the repo locally from provided project and repository name
```

Comment:
I would make this step optional, what do you think?

Reply:
Like the step where I check for GIT installed? I need it later on when checking if the repo contains required branch (develop). I will use it later on to create default CI if not found. It asks you, if you want to install git before the installation. WDYT is this ok?

Reply:
Another option could be to ask a user for git local path.

Reply:
now I get it what is behind - you have to have the local repo to set the policies, right? 
Otherwise you may want just to update the policies and not download anything (like developer does not have rights and he has to ask someone with the rights to do this step)

Reply:
yep. Please resolve the conversation if it makes sense to you